samples/app-cache-meetings/nodejs/src/components/app-cache-tab.tsx

```diff
@@ -0,0 +1,112 @@
+// <copyright file="app-cache-tab.tsx" company="Microsoft Corporation">
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+// </copyright>
+
+import { useState } from "react";
+import React from "react";
+import * as microsoftTeams from "@microsoft/teams-js";
+
+/// </summary>
+/// function logitem
+/// </summary>
+function logItem(action: string, actionColor: string, message: string) {
+  const newItem =
+    "<span style='font-weight:bold;color:" +
+    actionColor +
+    "'>" +
+    action +
+    "</span> " +
+    message +
+    "</br>";
+  return newItem;
+}
+
+/// </summary>
+/// In beforeUnloadHandler using setItems and readyToUnload callback function
+/// </summary>
+const beforeUnloadHandler = (
+  readyToUnload: () => void) => {
+  console.log("sending readyToUnload to TEAMS");
+  readyToUnload();
+
+  return true;
+};
+
+/// </summary>
+/// loadHandler using setItems to set values
+/// </summary>
+const loadHandler = (
+    setItems: React.Dispatch<React.SetStateAction<string[]>>,
+    data: microsoftTeams.LoadContext) => {
+    logItem("OnLoad", "blue", "Started for " + data.entityId);
+
+    let newItem = logItem("OnLoad", "blue", "Completed for " + data.entityId);
+    setItems((Items) => [...Items, newItem]);
+
+    microsoftTeams.app.notifySuccess();
+};
+
+const AppCacheTab = () => {
```

Comment:
the screenshots gow back to back load messages. Something is wrong. Are you not seeing the unload handler invoked? unlod for previous instance of the Tab needs to complete before load for the new instance happens

Reply:
also some screenshots show a blank entity id. pls remove if that is old/outdated